gamePlay.js

```diff
@@ -0,0 +1,132 @@
+var fs = require("fs");
+
+var inquirer = require('inquirer');
+
+var Word = require('./Word');
+
+var Letter = require("./Letter");
+
+var prompt = require('prompt');
+ 
+var triesLeft = 10;
+
+var choosenWord;
+
+var wordArray =[];
+
+var wordString;
+
+var noCount;
+
+var guessLeft = 5;
+
+var wrongString;
+
+var wrongArray = [];
+
+var correctCount;
+
+function start(){
+  wrongArray = [];
+  wordArray = [];
+  wordString = '';
+  wrongString = '';
+  noCount = 0;
+  guessLeft = 5;
+  correctCount = 0;
+  choosenWord = new Word();
+  for (var i=0; i< choosenWord.word.length; i++) {
+    wordArray[i] = "_";
+  }
+  wordString = wordArray.join(' ');
+  console.log (wordString);
+  userGuess();
+};
+
+function userGuess() {
+  inquirer.prompt([
+
+  {
+    type: "input",
+    message: "Guess a letter!",
+    name: "guess"
+  }
+
+  ]).then(function(answer) {
+      for (var i = 0; i < choosenWord.letters.length; i++) {
+        if (answer.guess == choosenWord.letters[i] ) {
+          wordArray[i] = answer.guess;
+          wordString = wordArray.join(' ');
```

Comment:
strings are tracked with indices as well so you don't need to refer to the word array and then join it.